src/BenchmarksApps/Grpc/GoClient/main.go

```diff
@@ -188,7 +188,7 @@ func calculateRequestStatistics() {
 	}
 	measurements := []jobMeasurement{
 		{
-			Name:      "grpc/rps/max",
+			Name:      "grpc/rps/mean",
```

Comment:
```suggestion
			Name:      "http/rps/mean",
```

Reply:
This is assuming that we want to have the same metrics as for http so we can see them on the same charts.